openvdb/points/PointGroup.h

```diff
@@ -641,14 +521,14 @@ inline void compactGroups(PointDataTree& tree)
         LeafManagerT leafManager(tree);
         tbb::parallel_for(leafManager.leafRange(), copy);
 
-        descriptor->setGroup(sourceName, targetOffset);
+        descriptor->setGroup(sourceName, targetOffset, /*checkValidOffset=*/true);
```

Comment:
Why does `checkValidOffset` need to be true here - doesn't this algorithm guarantee this?

Reply:
I believe it to be just an abundance of caution. I've addressed all feedback now.